modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -67,84 +64,68 @@ export default class Tile3DLayer extends CompositeLayer {
     return info;
   }
 
-  async _loadTileset(tilesetUrl, fetchOptions, ionMetadata) {
-    const response = await fetch(tilesetUrl, fetchOptions);
-    const tilesetJson = await response.json();
-
-    const loadOptions = this.getLoadOptions();
-
-    const tileset3d = new Tileset3D(tilesetJson, tilesetUrl, {
-      onTileLoad: tileHeader => {
-        this.props.onTileLoad(tileHeader);
-        this._updateTileset(tileset3d);
-      },
+  async _loadTileset(tilesetUrl) {
+    const {loader, loadOptions} = this.props;
+    const tilesetJson = await load(tilesetUrl, loader, loadOptions);
+    const tileset3d = new Tileset3D(tilesetJson, {
+      onTileLoad: this._onTileLoad.bind(this),
       onTileUnload: this.props.onTileUnload,
       onTileLoadFail: this.props.onTileError,
-      // TODO: explicit passing should not be needed, registerLoaders should suffice
-      fetchOptions,
-      ...ionMetadata,
-      ...loadOptions
+      fetchOptions: loadOptions
     });
```

Comment:
So users will not be able to set any tiles loader options other than `fetch`?

Reply:
I'll update this, should allow user to pass in other loadOptions